locale/en/docs/guides/event-loop-timers-and-nexttick.md

```diff
@@ -76,12 +76,14 @@ actually uses - are those above._
 
 * **timers**: this phase executes callbacks scheduled by `setTimeout()`
  and `setInterval()`.
-* **I/O callbacks**: executes almost all callbacks with the exception of
- close callbacks, the ones scheduled by timers, and `setImmediate()`.
+* **pending callbacks**: executes I/O callbacks deferred to the next loop
+ iteration.
 * **idle, prepare**: only used internally.
-* **poll**: retrieve new I/O events; node will block here when appropriate.
+* **poll**: retrieve new I/O events; execute I/O related callbacks (almost
+ all with the exception of close callbacks, the ones scheduled by timers,
+ and `setImmediate()`); node will block here when appropriate.
 * **check**: `setImmediate()` callbacks are invoked here.
-* **close callbacks**: e.g. `socket.on('close', ...)`.
+* **close callbacks**: some close callbacks, e.g. `socket.on('close', ...)`.
```

Comment:
with or without "some close callbacks" are both ok :)

Reply:
My worry is that someone would never read the detail and assume that all close callbacks are executed in this phase. Here's what the detail says:

> If a socket or handle is closed abruptly (e.g. socket.destroy()), the 'close' event will be emitted in this phase. Otherwise it will be emitted via process.nextTick().

So it's actually via `process.nextTick()` (or after any phase) where *most* close callbacks will be executed. Adding "some close callbacks" should lead folks to read the details if they want to know more.